osu.Game/Overlays/Changelog/StreamBadge.cs

```diff
@@ -64,7 +64,7 @@ public StreamBadge(APIChangelog changelogEntry)
                         new SpriteText
                         {
                             Text = ChangelogEntry.Users > 0 ?
-                                string.Join(" ", ChangelogEntry.Users.ToString("N0"), "users online"):
+                                string.Format($"{ChangelogEntry.Users:N0} users online") :
```

Comment:
No need for `string.Format`, the interpolated string does the job by itself.

Reply:
Appveyor doesn't seem happy without it
![image](https://user-images.githubusercontent.com/16261733/42993982-2aee4c12-8c0d-11e8-9ca8-b0e164ff3272.png)

Reply:
Should I ignore it?

Reply:
I have seen that error too, and I just specified the formatting culture, so using `.ToString("N0", CultureInfo.InvariantCulture)` is probably what it needs

Reply:
InvariantCulture breaks formatting for other languages. Right now I have that number nicely formatted in my language, using InvariantCulture sets everything to English. Are we doing this?